internal/dag/ingress_processor.go

```diff
@@ -154,13 +154,11 @@ func (p *IngressProcessor) computeIngressRule(ing *networking_v1.Ingress, rule n
 		be := httppath.Backend
 		m := types.NamespacedName{Name: be.Service.Name, Namespace: ing.Namespace}
 
-		var port intstr.IntOrString
+		port := int(be.Service.Port.Number)
 		if len(be.Service.Port.Name) > 0 {
-			port = intstr.FromString(be.Service.Port.Name)
-		} else {
-			port = intstr.FromInt(int(be.Service.Port.Number))
+			_, svcPort, _ := p.source.LookupService(m, intstr.FromString(be.Service.Port.Name))
```

Comment:
we need to handle the error here

Reply:
If there is an error, the `svcPort`'s Port will be 0, it will lead to `EnsureService` returns a error, so  ignore it here

Reply:
ok